src/components/ControlledInputFields/ControlledTextInputField/ControlledTextInputField.js

```diff
@@ -23,18 +23,18 @@ const useStyles = makeStyles((theme) => ({
 }))
 
 const ControlledTextInputField = ({
+  isRequired = false,
   control,
   name,
   label,
-  placeholder,
-  customValidationFunctions,
+  placeholder = "",
```

Comment:
Remember what we discussed about prop forwarding? Seems like you should be making use of that here